### PR TITLE
tailwind-theme

### PR DESCRIPTION
**Pull request by kikavlntn**

### CODE REVIEW

**Comment by kikavlntn** [comment]
issue #1 solved

File: tailwind.config.cjs [hunk 1]
```diff
@@ -2,6 +2,19 @@
 module.exports = {
   content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
+    fontFamily: {
+      default: ["Inter"],
+    },
+    colors: {
+      bg: "#1C1C27",
```
Comment (dnmct) [inline tailwind.config.cjs:9]: Group colors that belong together in an object
(e.g. all the whites and all the darks)
https://tailwindcss.com/docs/customizing-colors#color-object-syntax

Comment (dnmct) [inline tailwind.config.cjs:9]: bg-bg is a weird name maybe use dark instead
its not just used for the background

Comment (amyria3) [reply tailwind.config.cjs:9]: i would then like to adjust the name in the figma color theme but I am not permitted. Could You do so?

Comment (dnmct) [reply tailwind.config.cjs]: done

File: tailwind.config.cjs [hunk 2]
```diff
@@ -6,12 +6,14 @@ module.exports = {
       default: ["Inter"],
     },
     colors: {
-      bg: "#1C1C27",
-      "bg-light": "#363740",
+      dark: "#1C1C27",
+      "dark-light": "#363740",
+      white: {
+        default: "#FFFFFF",
```
Comment (dnmct) [inline tailwind.config.cjs:12]: needs to be DEFAULT

File: tailwind.config.cjs [hunk 3]
```diff
@@ -6,12 +6,14 @@ module.exports = {
       default: ["Inter"],
     },
     colors: {
-      bg: "#1C1C27",
-      "bg-light": "#363740",
+      dark: "#1C1C27",
```
Comment (dnmct) [inline tailwind.config.cjs:9]: dark is still not grouped

**Review by dnmct — CHANGES_REQUESTED** [review]

File: tailwind.config.cjs [hunk 4]
```diff
@@ -2,6 +2,23 @@
 module.exports = {
   content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
+    fontFamily: {
+      default: ["Inter"],
```
Comment (ReallyBoard123) [inline tailwind.config.cjs:6]: Make the default in CAPITAL

File: src/main.tsx [hunk 5]
```diff
@@ -3,5 +3,7 @@ import ReactDOM from "react-dom/client";
 import "./index.css";
 
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
-  <React.StrictMode>{/* Your code goes here... */}</React.StrictMode>
+  <React.StrictMode>
+    <h1 className="text-title bg-">hello world</h1>
```
Comment (ReallyBoard123) [inline src/main.tsx:7]: is bg- missing a value here? or just remove it

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
see comments, good job!

File: tailwind.config.cjs (hunk 4 above)
Comment (kikavlntn) [reply tailwind.config.cjs:6]: okis

File: tailwind.config.cjs (hunk 2 above)
Comment (kikavlntn) [reply tailwind.config.cjs]: done

**Review by ReallyBoard123 — APPROVED** [review]

**Review by dnmct — APPROVED** [review]

**Pull request merged**